packages/async-operations/src/index.ts

```diff
@@ -0,0 +1,2 @@
+import { startAsyncOperation } from './async_operations';
+module.exports = { startAsyncOperation };
```

Comment:
This is a bit more than sub-optimal, but to stub out startAsyncOperation, we need to continue to use CommonJS exports or refactor significantly.    See slack discussion.

Reply:
This was discussed and agreed that properly fixing the API endpoint test/logic/etc exceeds the scope for this ticket (see CUMULUS-2260 ) -- if there's an obvious proxyquire/setup for the tests that can be done cleanly in the API endpoints without significant refactor, I'm all 👂

Reply:
Agreed, I'm deeply unhappy with doing that.